### ISSUE TITLE
pattern for abnormal response to entity

### ISSUE THREAD

**Issue by pfey03** [obophenotype/upheno#754]
I have many abnormal response to terms to chemical entities but also sometimes to a biological process

PATO_0000077 response to

AbnormalResponseToEntity

Maybe also children in/decreased/absent

**Comment by matentzn** [comment]
That seems very general - we generally prefer to have these pushed into GO (response to X terms) and then simply use the abnormalBiologicalProcess pattern. Can you give some examples where that would not work?

**Comment by mah11** [comment]
The issue here is that GO has been pushing back lately. I'd be happy to carry on using abnormalBiologicalProcess + GO response-to-X, but GO's trying to clear out a lot of "response to" terms, and not add many new ones, I think because they want to apply more stringent criteria for what they'll count as a real, naturally occurring, distinct response process. People like me & Petra, and our ontologies, are caught in the middle :/

I don't have an example of something currently un-model-able from FYPO off the top of my head, but there are a few that might lose LDs if GO makes a "response to" term obsolete AND we don't have an alternative phenotype pattern. FYPO:0001932 "abnormal cellular response to hydroxyurea" is one.

For more info, try [this GO ticket](https://github.com/geneontology/go-ontology/issues/14303), and some of the others with the "response_to_terms" label in that issue tracker.

**Comment by matentzn** [comment]
Ok thank you @mah11 

One option would be to create a response to exposure to entity pattern. It would look convoluted, but it is actually quite easy in the end, something like:

```
GO:response to and 'has stimulus' some 'CHEBI:carbon' and 'has route' some 'injection'
```

How does GO model the relation between "response to X" processes and the X?

**Comment by mah11** [comment]
> How does GO model the relation between "response to X" processes and the X?

I don't know, and I don't see anything obvious in https://github.com/geneontology/go-ontology/tree/master/src/design_patterns or https://github.com/geneontology/go-ontology/tree/master/docs/patterns ... best ask someone there, e.g. @pgaudet

**Comment by pfey03** [comment]
@mah11 yes I also feel GO gets rather rid of useful terms for phenotypes, I went through our whole ontology and there were terms to previous GO terms but this have been removed from GO. 

We have mostly response to chemicals/CHEBI and a few to processes (most of those are in GO). Examples:

aberrant response to N-Phenylthiourea
aberrant response to SDF-1
abolished response to cheating def: "Complete resistance to cheating when mixed with cheater cells." (GO:0099139:cheating during chimeric sorocarp development is Exact synonym)

**Comment by pgaudet** [comment]
Hello, 

We use for example
GO:0033198 response to ATP

'response to stimulus'
 and ('has input' some 'ATP(4-)')

Thanks, Pascale

**Comment by dosumis** [comment]
Update - there is a concerted effort on the equivalent Trait patterns for OBA (CC @kallia-p).  

We should have w patterns:

abnormal_response_to_chemical

E = GO:'response to chemical' and has_input some CHEBI:{some chemical}
Q = PATO:'process quality'
var = chemical

abnormal_response_to_stress
var = E = subclass of of GO:'response to stress'

Where E & Q follow the general pattern:
has_part some (Q and 'characteristic of' some E) and (has_modifier some 'abnormal')
